### PR TITLE
[AA-1737] Update Admin App to .NET 8

### PR DESCRIPTION
**Pull request by jleiva-gap**
- Upgrade project to NET 8
- Upgrade nugets
- Upgrade Ed-Fi packages
- Update EF code to apply changes added to EF Core 8
- Fix tests

### CODE REVIEW

File: Application/EdFi.Ods.AdminApp.Management.Tests/MockExtensions.cs [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+// SPDX-License-Identifier: Apache-2.0
 // Licensed to the Ed-Fi Alliance under one or more agreements.
 // The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 // See the LICENSE and NOTICES files in the project root for more information.
 
 using System;
 using System.Collections.Generic;
-using System.Data.Entity;
-using System.Data.Entity.Infrastructure;
+//using System.Data.Entity;
```
Comment (DavidJGapCR) [inline Application/EdFi.Ods.AdminApp.Management.Tests/MockExtensions.cs:8]: Can we remove these lines?

File: Application/EdFi.Ods.AdminApp.Management/ClaimSetEditor/CopyClaimSetCommandV6Service.cs [hunk 2]
```diff
@@ -4,10 +4,11 @@
 // See the LICENSE and NOTICES files in the project root for more information.
 
 using System.Collections.Generic;
-using System.Data.Entity;
+//using System.Data.Entity;
```
Comment (DavidJGapCR) [inline Application/EdFi.Ods.AdminApp.Management/ClaimSetEditor/CopyClaimSetCommandV6Service.cs:7]: Same here

File: Application/EdFi.Ods.AdminApp.Management/ClaimSetEditor/CopyClaimSetCommandV53Service.cs [hunk 3]
```diff
@@ -2,13 +2,15 @@
 // Licensed to the Ed-Fi Alliance under one or more agreements.
 // The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 // See the LICENSE and NOTICES files in the project root for more information.
+extern alias SecurityCompatiblity53;
 
-using System.Data.Entity;
+//using System.Data.Entity;
```
Comment (jagudelo-gap) [inline Application/EdFi.Ods.AdminApp.Management/ClaimSetEditor/CopyClaimSetCommandV53Service.cs]: Remove line

File: Application/EdFi.Ods.AdminApp.Management/IFirstTimeSetupService.cs [hunk 4]
```diff
@@ -48,9 +48,11 @@ protected FirstTimeSetupServiceBase(IUsersContext usersContext, ISecurePackedHas
         {
             var applicationName = odsInstanceName.GetAdminApplicationName();
 
-            var existingApplication = await UsersContext.Applications.SingleOrDefaultAsync(x =>
+            var existingApplication = UsersContext.Applications.AsEnumerable().FirstOrDefault(x =>
+                x.ApplicationName.ToUpper() == applicationName.ToUpper());
+            /*var existingApplication = await UsersContext.Applications.AsEnumerable().FirstOrDefaultAsync(x =>
```
Comment (jagudelo-gap) [inline Application/EdFi.Ods.AdminApp.Management/IFirstTimeSetupService.cs]: Remove comment out lines

File: Application/EdFi.Ods.AdminApp.Web/Startup.cs [hunk 5]
```diff
@@ -41,9 +40,10 @@
 using NUglify.JavaScript;
 using System.Security.Claims;
 using System.Linq;
-using EdFi.Admin.DataAccess.DbConfigurations;
+// using EdFi.Admin.DataAccess.DbConfigurations;
```
Comment (jagudelo-gap) [inline Application/EdFi.Ods.AdminApp.Web/Startup.cs]: remove line

File: Application/EdFi.Ods.AdminApp.Management.Tests/TestDbAsyncEnumerator.cs [hunk 6]
```diff
@@ -64,14 +71,19 @@ public TestDbAsyncEnumerable(Expression expression)
...
-        IDbAsyncEnumerator IDbAsyncEnumerable.GetAsyncEnumerator()
+        IAsyncEnumerator<T> IAsyncEnumerable<T>.GetAsyncEnumerator(CancellationToken cancellationToken)
         {
-            return GetAsyncEnumerator();
+            return GetAsyncEnumerator(cancellationToken);
+        }
+
+        public IAsyncEnumerator<T> GetAsyncEnumerator(CancellationToken cancellationToken = default)
+        {
+            return  GetAsyncEnumerator(cancellationToken);  //new TestDbAsyncEnumerator<T>(this.AsEnumerable().GetEnumerator());
```
Comment (jagudelo-gap) [inline Application/EdFi.Ods.AdminApp.Management.Tests/TestDbAsyncEnumerator.cs]: remove comment out line

**Review by jagudelo-gap — CHANGES_REQUESTED** [review]

**Review by jagudelo-gap — APPROVED** [review]

**Pull request merged**